ci/cached-builds/make_test.py

```diff
@@ -19,13 +19,14 @@
 
 class Args(argparse.Namespace):
     """Type annotation to have autocompletion for args"""
+
     target: str
 
 
 def main() -> None:
     parser = argparse.ArgumentParser("make_test.py")
     parser.add_argument("--target", type=str)
-    args = typing.cast(Args, parser.parse_args())
+    args = typing.cast("Args", parser.parse_args())
```

Comment:
Hm, is this truly harmless? Maybe yes, it just seems weird to me.

Reply:
https://docs.astral.sh/ruff/rules/runtime-cast-value/#why-is-this-bad

I did not think about this, but they're right.